src/AutoRest.CSharp/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -90,7 +90,8 @@ private static Configuration LoadConfiguration(string basePath, string json)
                 root.GetProperty(nameof(Configuration.PublicClients)).GetBoolean(),
                 root.GetProperty(nameof(Configuration.ModelNamespace)).GetBoolean(),
                 root.GetProperty(nameof(Configuration.HeadAsBoolean)).GetBoolean(),
-                root.GetProperty(nameof(Configuration.SkipCSProjPackageReference)).GetBoolean()
+                root.GetProperty(nameof(Configuration.SkipCSProjPackageReference)).GetBoolean(),
+                root.GetProperty(nameof(Configuration.OperationGroupMapping))
```

Comment:
You're reading `OperationGroupMapping` property but not writing [here](https://github.com/Azure/autorest.csharp/blob/3566a512d973f526805c99aa5bdb48e744c1caf5/src/AutoRest.CSharp/AutoRest/Communication/StandaloneGeneratorRunner.cs#L59). I think If you don't write it, it will throw an error when you will do F5 for any project.

Reply:
Updated